addons/l10n_tw_chart/__openerp__.py

```diff
@@ -0,0 +1,28 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+# Copyright (C) 2014-now  Jason Wu (jaronemo@msn.com)
+
+{
+    'name': '台灣 國際財務報導準則(IFRS)之會計科(項)目表',
+    'version': '1.0',
+    'category': 'Localization/Account Charts',
+    'author': 'Jason Wu',
+    'maintainer': 'Jason Wu(jaronemo@msn.com)',
+    'website': 'http://www.osstw.com',
+    'description': """
+
+    參照  2016 / 05 台灣 國際財務報導準則(IFRS)之會計科(項)目表 更新事項
```

Comment:
Having Taiwanese in the description is fine but having a small description for English speakers too is advised.